rosa_learning/creating_cluster_workshop/cloud-experts-getting-started-deleting.adoc

```diff
@@ -0,0 +1,91 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-deleting"]
+= Tutorial: Deleting your cluster
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-deleting
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-01-11
+
+You can delete your {product-title} (ROSA) cluster using either the command line interface (CLI) or the user interface (UI). 
+
+== Deleting a ROSA cluster using the CLI
+
+. *Optional:* List your clusters to make sure you are deleting the correct one by running the following command:
++
+[source,terminal]
+----
+$ rosa list clusters
+----
+
+. Delete a cluster by running the following command:
++
+[source,terminal]
+----
+$ rosa delete cluster --cluster <cluster-name>
+----
++
+[WARNING]
+====
+This command is non-recoverable.
+====
+
+. The CLI prompts you to confirm that you want to delete the cluster. Press *y* and then *Enter*. The cluster and all its associated infrastructure will be deleted.
++
+[NOTE]
+====
+All AWS STS and IAM roles and policies will remain and must be deleted manually once the cluster deletion is complete by following the steps below.
```

Comment:
@yingzhanredhat Can you please confirm this part? If HCP cluster deletion mentioned `the policies will remain, follow xxx to delete xxx` then let's file a bug. HCP is using managed policies those cannot be deleted

Reply:
@xueli181114 It doesn't show this message. It just provide the operator-roles/oidc-provider deleted steps